src/components/atoms/FreeTermHour/FreeTermHour.tsx

```diff
@@ -2,24 +2,16 @@ import React, { useState } from "react";
 import { FreeTerm } from "types";
 import "./FreeTermHour.css";
 import { Confirm } from "../../molecules/Confirm/Confirm";
+import { sendData } from "../../../api";
 
 export const FreeTermHour = (props: FreeTerm) => {
   const [display, setDisplay] = useState(false);
   const [free, setFree] = useState(false);
 
   const id: string = props.id;
 
-  const bookTerm = async () => {
-    await fetch("http://localhost:3001/term/book-term", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({
-        id,
-      }),
-    });
-
+  const bookTerm = () => {
+    sendData(id, "http://localhost:3001/term/book-term");
```

Comment:
Mniej więcej o to chodziło. Adresy url powinny być trzymane inaczej. BASE_URL czyli w tym wypadku "http://localhost:3001" powinno być w `.env`, a pozostała część adresu powinno być w zmiennej albo enum:
```
const routes = {
  bookTerm: '/term/book-term',
  ...
};
```
i ten plik też powinien być trzymany gdzieś w okolicach pliku api.ts.